src/utils/interfaces.ts

```diff
@@ -1,11 +1,12 @@
 // This is a file of TS interfaces that get used in multiple places
 // So they live here so you can import them!
 
-interface Card {
+interface CardObj {
```

Comment:
this had to be CardObj instead of just 'Card' because in the Card component, we were using two Card names and typescript didnt like it